src/components/LocationTime/LocationTime.js

```diff
@@ -1,12 +1,33 @@
+import { DateTime } from "luxon";
+import { useState } from "react";
 import "./LocationTime.css";
 
 export function LocationTime({
-    time,
     location,
-    day,
-    date,
-    alignment
+    timezone = "local",
+    alignment = "left",
 }) {
+    let time = null;
+    let day = null;
+    let date = null;
+    
+    // By default, as local timezone is selected, we create date in local time
+    // let dt = DateTime.local();
+    let [ dt, setDateTime ] = useState(DateTime.local());
+
+    // For timezones other than local
+    if(timezone !== "local") {
+        dt = dt.setZone(timezone);
+    }
+
+    setTimeout(() => {
+        setDateTime(DateTime.local());
+    }, 1000);
+
+    time = dt.toLocaleString(DateTime.TIME_24_SIMPLE);
+    day = dt.toLocaleString({ weekday: 'long' });
+    date = dt.toLocaleString({ day: 'numeric', month: 'short' })
+
     return(
         <div className={ alignment && alignment === "right" ? "location-time flex-align-end" : "location-time flex-align-start" }>
```

Comment:
alignment && not required

`{location-time ${alignment === "right" ? "flex-align-end" : "flex-align-start"}}`

Reply:
Updated